### ISSUE TITLE
Allow to sort by params

### ISSUE THREAD

**Issue by mateuszpieniak** [polyaxon/polyaxon#647]
### Feature motivation

I would like to see how the sampled hyperparameters affect the validation score. It gives a grasp of information about hyperparameter space. In addition, I noticed some Polyaxon's sampling bug that could be easier detected with such a feature.

### Feature description

Allow sorting experiment by values from parameter columns.

The feature is already implemented for metric columns.

**Comment by mouradmourafiq** [comment]
It's possible to do that, you just need to add that column to the table and the order by button will show the param option, then you can order by asc/desc by that param.

**Comment by mateuszpieniak** [comment]
It doesn't show the learning rate as you can see below.

![image](https://user-images.githubusercontent.com/31375424/71994228-acfc6180-3238-11ea-9ee0-317122df6527.png)

**Comment by mouradmourafiq** [comment]
this is probably because the dots in the name? did you try another param to see if it works?

**Comment by mateuszpieniak** [comment]
I just tried with some artificial test param

```
hptuning:
  concurrency: 4
  random_search:
    n_experiments: 30
  matrix:
    test:
        values: [1, 2, 3]
```

but sorting is still not available.

**Comment by mouradmourafiq** [comment]
This is strange, this sounds like a bug.

**Comment by mateuszpieniak** [comment]
It looks like, but I added that as a feature request because I didn't know whether it was done on purpose.

**Comment by mouradmourafiq** [comment]
I will check further, there must be an issue somewhere. Just to make sure when you started the new hp tuning group, did you try to sort the table on the default experiments page or the table on the group overview page?

**Comment by mateuszpieniak** [comment]
Group overview page.

**Comment by mouradmourafiq** [comment]
ok, there's a patch release planned for end of this week, I will try to fix this issue as well. Thanks for reporting.

**Comment by mouradmourafiq** [comment]
I was confused, this functionality was not developed in the v0.x, I will mark it as feature.

**Comment by mouradmourafiq** [comment]
I just remembered why this feature was not implemented in v0.x (I also forgot this argument, and that's why it was implement in v1.x). The idea of not providing order by for params is that they are discrete values and will be repetitive for several experiments, you can get all those experiments  by using a filter instead in the search bar for that specific param(s).

I am closing this issue, as I don't think this will be implemented in v0.x (the filter workaround should get you the experiments list), but the feature is implemented in v1.x, and can be used once v1.x is available to CE users.
